### PR TITLE
add new !Cidr function

### PR DESCRIPTION
**Pull request by tyron**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to the "aws-cloudformation-yaml" extension will be documente
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [0.1.3] - 2018.03.09
+
+### Added
+
+* Intrinsic function snippets
+  * !GetCidr
```
Comment (bitfoo1) [inline CHANGELOG.md]: !GetCidr -> !Cidr
As the original function in CFN.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@ Following Snippets do provide this extension for YAML files:
     * __!FindInMap__
     * __!GetAtt__
     * __!GetAZs__
+    * __!GetCidr__
```
Comment (bitfoo1) [inline README.md]: __!GetCidr__ -> !Cidr
See above.

**Review by bitfoo1 — CHANGES_REQUESTED** [review]
@tyron thank you for adding this awesome function. But please would you change mentioned comments. And maybe you are able to rebase to only one commit?
Thank you!

**Comment by tyron** [comment]
@dthielking I made the changes requested. However my very limited proficiency using git doesn't allow me to rebase to one commit. I failed in my attempts to squash it on my local machine.

**Review by bitfoo1 — APPROVED** [review]

**Comment by bitfoo1** [comment]
@tyron thank you for fixing the documentation miss match. I squash merged this branch into master and will create new version.

**Pull request merged**